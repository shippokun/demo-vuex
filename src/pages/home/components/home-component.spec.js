import { shallowMount } from "@vue/test-utils";
import HomeComponent from "./home-component";

describe("home-component.vue", () => {
  it("renders props.msg when passed", () => {
    const name = "test name";
    const wrapper = shallowMount(HomeComponent, { propsData: { name } });
    expect(wrapper.text()).toMatch(name);
  });
});
