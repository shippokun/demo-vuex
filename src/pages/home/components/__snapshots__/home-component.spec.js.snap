// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home-component.vue renders props.msg when passed 1`] = `
<div class="home">
  <h1>test name</h1>
</div>
`;
